src/gui/mainclient.cpp

```diff
@@ -54,22 +55,42 @@ void warnSystray() {
 }
 
 void signalHandler(int signum) {
-    fprintf(stderr, "Server stoped with signal %d\n", signum);
+    fprintf(stderr, "Client stoped with signal %d\n", signum);
 
-    // Make sure everything flushes
-#ifdef NDEBUG
-    auto sentryClose = qScopeGuard([] { sentry_close(); });
+    KDC::SyncPath sigFilePath = std::filesystem::temp_directory_path();
+    if (signum == SIGSEGV || signum == SIGFPE || signum == SIGILL
+#ifndef Q_OS_WIN
+        || signum == SIGBUS
 #endif
+    ) {
+        // Crash
+        sigFilePath /= KDC::crashClientFileName;
+    } else {
+        // Kill
+        sigFilePath /= KDC::killClientFileName;
+    }
+
+    std::ofstream sigFile(sigFilePath);
+    if (sigFile) {
+        sigFile << signum << std::endl;
```

Comment:
For debugging purpose,  a string would be nicer to read.

Reply:
This file is temporary and its content is read by the app, it does not seem wise to me to write a label there instead of a code.

Reply:
Maybe I don't understand the intent. Here is what I am thinking about.

Your code has been merged and is now deployed. The Support Team reports a problem that we don't immediately understand. Quite encouragingly, the user has found a file on  her disk with a suggestive name (`sigFilePath`) whose content is `42`.
Wouldn't it better to read `SIGABORT` instead, or some text that describes accurately the signal at play? 
The situation I am alluding to is similar to the logging of integral error codes. If we know them by heart, then it's fine. But we usually don't.

Reply:
I don't agree. It is a technical file, it is simpler and more efficient to write/read codes in it rather than labels (like in a csv file for example).
And it is easy to find what a signal number corresponds to (man signal).

Reply:
We disagree, that's fine. Resolving.